javaHapiValidatorLambda/serverless.yml

```diff
@@ -23,16 +23,29 @@ provider:
   environment:
     FHIR_VERSION:
       !Ref FhirVersion
-
+    IMPLEMENTATION_GUIDES_BUCKET: !ImportValue ImplementationGuidesBucketNameKey
+  iam:
```

Comment:
Put the role on the function level? If we add more functions to this stack in the future, it'll be automatically applied to the new function and might create confusion.